packages/types/src/interfaces/injects.ts

```diff
@@ -62,6 +62,51 @@ export const typesBundle: OverrideBundleType = {
       ],
     },
   },
+  // chain: {
```

Comment:
Tested the script when we use specific types for different chains and it gives me the right types for right genesis hash..
````
typesInChain::  {
  '0xba29ccef64182e17dee0f9d8bbaddc69e439acdc9409149e5c409d696c14232e': {
    types: {
      AttestationTopic: 'u256',
      AttestationValue: 'u256',
      ExchangeKey: '(AssetId, AssetId)',
      FeeRate: 'u128',
      LiquidityValueResponse: [Object],
      PriceResponse: [Object],
      LiquidityPriceResponse: [Object],
      AssetOptions: [Object],
      AssetInfo: [Object],
      Owner: [Object],
      PermissionsV1: [Object],
      PermissionVersions: [Object],
      PermissionLatest: 'PermissionsV1',
      WithdrawReasons: [Object],
      BalanceLock: [Object],
      TokenId: '(CollectionId, SeriesId, SerialNumber)',
      AuctionClosureReason: [Object],
      AuctionListing: [Object],
      CollectionId: 'u32',
      CollectionNameType: 'Vec<u8>',
      FixedPriceListing: [Object],
      Listing: [Object],
      ListingId: 'u128',
      MetadataBaseURI: [Object],
      NFTAttributeValue: [Object],
      Reason: 'AuctionClosureReason',
      RoyaltiesSchedule: [Object],
      SeriesId: 'u32',
      SerialNumber: 'u32',
      TokenCount: 'u32',
      RewardBalance: 'Balance',
      RewardBalanceOf: 'Balance',
      RewardDestination: [Object],
      VecDeque: 'Vec<u64>',
      Meta: 'Vec<(Text, Text)>',
      MemberRoles: [Object],
      Member: [Object],
      Group: [Object],
      Invite: [Object],
      PendingInvite: [Object],
      AcceptPayload: [Object],
      DeviceId: 'u32',
      PreKeyBundle: 'Bytes',
      DeviceIdResponse: 'DeviceId',
      WithdrawnPreKeyBundle: '(AccountId, u32, Bytes)',
      PreKeyBundlesResponse: 'Vec<WithdrawnPreKeyBundle>',
      Response: [Object],
      VaultKey: 'Bytes',
      VaultValue: 'Bytes',
      MessageId: 'u32',
      Message: 'Bytes',
      Address: 'AccountId',
      Index: 'u64',
      doughnut: 'Raw',
      PhaseTo36: [Object],
      DispatchClassTo36: [Object],
      WeightTo36: 'u32',
      DispatchInfoTo36: [Object],
      FeeExchangeV1: [Object],
      FeeExchange: [Object],
      ChargeTransactionPayment: [Object]
    },
    userExtensions: {
      ChargeTransactionPayment: [Object],
      CheckEra: [Object],
      CheckGenesis: [Object],
      CheckNonce: [Object],
      CheckSpecVersion: [Object],
      CheckTxVersion: [Object]
    }
  },
  '0x0d0971c150a9741b8719b3c6c9c2e96ec5b2e3fb83641af868e6650f3e263ef0': {
    types: {
      Food: [Object],
      AttestationTopic: 'u256',
      AttestationValue: 'u256',
      ExchangeKey: '(AssetId, AssetId)',
      FeeRate: 'u128',
      LiquidityValueResponse: [Object],
      PriceResponse: [Object],
      LiquidityPriceResponse: [Object],
      AssetOptions: [Object],
      AssetInfo: [Object],
      Owner: [Object],
      PermissionsV1: [Object],
      PermissionVersions: [Object],
      PermissionLatest: 'PermissionsV1',
      WithdrawReasons: [Object],
      BalanceLock: [Object],
      TokenId: '(CollectionId, SeriesId, SerialNumber)',
      AuctionClosureReason: [Object],
      AuctionListing: [Object],
      CollectionId: 'u32',
      CollectionNameType: 'Vec<u8>',
      FixedPriceListing: [Object],
      Listing: [Object],
      ListingId: 'u128',
      MetadataBaseURI: [Object],
      NFTAttributeValue: [Object],
      Reason: 'AuctionClosureReason',
      RoyaltiesSchedule: [Object],
      SeriesId: 'u32',
      SerialNumber: 'u32',
      TokenCount: 'u32',
      RewardBalance: 'Balance',
      RewardBalanceOf: 'Balance',
      RewardDestination: [Object],
      VecDeque: 'Vec<u64>',
      Meta: 'Vec<(Text, Text)>',
      MemberRoles: [Object],
      Member: [Object],
      Group: [Object],
      Invite: [Object],
      PendingInvite: [Object],
      AcceptPayload: [Object],
      DeviceId: 'u32',
      PreKeyBundle: 'Bytes',
      DeviceIdResponse: 'DeviceId',
      WithdrawnPreKeyBundle: '(AccountId, u32, Bytes)',
      PreKeyBundlesResponse: 'Vec<WithdrawnPreKeyBundle>',
      Response: [Object],
      VaultKey: 'Bytes',
      VaultValue: 'Bytes',
      MessageId: 'u32',
      Message: 'Bytes',
      Address: 'AccountId',
      Index: 'u64',
      doughnut: 'Raw',
      PhaseTo36: [Object],
      DispatchClassTo36: [Object],
      WeightTo36: 'u32',
      DispatchInfoTo36: [Object],
      FeeExchangeV1: [Object],
      FeeExchange: [Object],
      ChargeTransactionPayment: [Object]
    },
    userExtensions: {
      ChargeTransactionPayment: [Object],
      CheckEra: [Object],
      CheckGenesis: [Object],
      CheckNonce: [Object],
      CheckSpecVersion: [Object],
      CheckTxVersion: [Object]
    }
  },
  '0xc65170707265757d8a1fb8e039062286b8f0092f2984f5938588bd8e0f21ca2e': {
    types: {
      Direction: [Object],
      AttestationTopic: 'u256',
      AttestationValue: 'u256',
      ExchangeKey: '(AssetId, AssetId)',
      FeeRate: 'u128',
      LiquidityValueResponse: [Object],
      PriceResponse: [Object],
      LiquidityPriceResponse: [Object],
      AssetOptions: [Object],
      AssetInfo: [Object],
      Owner: [Object],
      PermissionsV1: [Object],
      PermissionVersions: [Object],
      PermissionLatest: 'PermissionsV1',
      WithdrawReasons: [Object],
      BalanceLock: [Object],
      TokenId: '(CollectionId, SeriesId, SerialNumber)',
      AuctionClosureReason: [Object],
      AuctionListing: [Object],
      CollectionId: 'u32',
      CollectionNameType: 'Vec<u8>',
      FixedPriceListing: [Object],
      Listing: [Object],
      ListingId: 'u128',
      MetadataBaseURI: [Object],
      NFTAttributeValue: [Object],
      Reason: 'AuctionClosureReason',
      RoyaltiesSchedule: [Object],
      SeriesId: 'u32',
      SerialNumber: 'u32',
      TokenCount: 'u32',
      RewardBalance: 'Balance',
      RewardBalanceOf: 'Balance',
      RewardDestination: [Object],
      VecDeque: 'Vec<u64>',
      Meta: 'Vec<(Text, Text)>',
      MemberRoles: [Object],
      Member: [Object],
      Group: [Object],
      Invite: [Object],
      PendingInvite: [Object],
      AcceptPayload: [Object],
      DeviceId: 'u32',
      PreKeyBundle: 'Bytes',
      DeviceIdResponse: 'DeviceId',
      WithdrawnPreKeyBundle: '(AccountId, u32, Bytes)',
      PreKeyBundlesResponse: 'Vec<WithdrawnPreKeyBundle>',
      Response: [Object],
      VaultKey: 'Bytes',
      VaultValue: 'Bytes',
      MessageId: 'u32',
      Message: 'Bytes',
      Address: 'AccountId',
      Index: 'u64',
      doughnut: 'Raw',
      PhaseTo36: [Object],
      DispatchClassTo36: [Object],
      WeightTo36: 'u32',
      DispatchInfoTo36: [Object],
      FeeExchangeV1: [Object],
      FeeExchange: [Object],
      ChargeTransactionPayment: [Object]
    },
    userExtensions: {
      ChargeTransactionPayment: [Object],
      CheckEra: [Object],
      CheckGenesis: [Object],
      CheckNonce: [Object],
      CheckSpecVersion: [Object],
      CheckTxVersion: [Object]
    }
  },
  '0x6f2ca0f5c770a9212d30b49604b2e45e2d3c949a3a940bfe908d68dbc4a0415e': {
    types: {
      WindowSize: [Object],
      AttestationTopic: 'u256',
      AttestationValue: 'u256',
      ExchangeKey: '(AssetId, AssetId)',
      FeeRate: 'u128',
      LiquidityValueResponse: [Object],
      PriceResponse: [Object],
      LiquidityPriceResponse: [Object],
      AssetOptions: [Object],
      AssetInfo: [Object],
      Owner: [Object],
      PermissionsV1: [Object],
      PermissionVersions: [Object],
      PermissionLatest: 'PermissionsV1',
      WithdrawReasons: [Object],
      BalanceLock: [Object],
      TokenId: '(CollectionId, SeriesId, SerialNumber)',
      AuctionClosureReason: [Object],
      AuctionListing: [Object],
      CollectionId: 'u32',
      CollectionNameType: 'Vec<u8>',
      FixedPriceListing: [Object],
      Listing: [Object],
      ListingId: 'u128',
      MetadataBaseURI: [Object],
      NFTAttributeValue: [Object],
      Reason: 'AuctionClosureReason',
      RoyaltiesSchedule: [Object],
      SeriesId: 'u32',
      SerialNumber: 'u32',
      TokenCount: 'u32',
      RewardBalance: 'Balance',
      RewardBalanceOf: 'Balance',
      RewardDestination: [Object],
      VecDeque: 'Vec<u64>',
      Meta: 'Vec<(Text, Text)>',
      MemberRoles: [Object],
      Member: [Object],
      Group: [Object],
      Invite: [Object],
      PendingInvite: [Object],
      AcceptPayload: [Object],
      DeviceId: 'u32',
      PreKeyBundle: 'Bytes',
      DeviceIdResponse: 'DeviceId',
      WithdrawnPreKeyBundle: '(AccountId, u32, Bytes)',
      PreKeyBundlesResponse: 'Vec<WithdrawnPreKeyBundle>',
      Response: [Object],
      VaultKey: 'Bytes',
      VaultValue: 'Bytes',
      MessageId: 'u32',
      Message: 'Bytes',
      Address: 'AccountId',
      Index: 'u64',
      doughnut: 'Raw',
      PhaseTo36: [Object],
      DispatchClassTo36: [Object],
      WeightTo36: 'u32',
      DispatchInfoTo36: [Object],
      FeeExchangeV1: [Object],
      FeeExchange: [Object],
      ChargeTransactionPayment: [Object]
    },
    userExtensions: {
      ChargeTransactionPayment: [Object],
      CheckEra: [Object],
      CheckGenesis: [Object],
      CheckNonce: [Object],
      CheckSpecVersion: [Object],
      CheckTxVersion: [Object]
    }
  }
}

````